css/css-writing-modes/inline-box-orthogonal-child-with-margins.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<link rel="author" title="Sergio Villar Senin" href="mailto:svillar@igalia.com"/>
+<link rel="help" href="https://drafts.csswg.org/css-writing-modes-3/#block-flow"
+  title="3.1 Block Flow Direction: the writing-mode property">
+<link rel="match" href="inline-box-orthogonal-child-with-margins-ref.html">
+ <style>
+.container {
+  border: 1px dashed blue;
+  height: 200px;
+  display: inline-block;
+}
+.container > div {
+  float: left;
+  width: 40px;
+  margin:       10px 20px 30px 40px;
+  border-width: 20px 30px 40px 50px;
+  padding:      30px 40px 50px 60px;
+  border-style: dotted;
+}
+.container > div:nth-child(1) { background: yellow; }
+.container > div:nth-child(2) { background: purple; }
+
+</style>
+
```

Comment:
It'd be nice to have the usual message, "the test passes if you see"; or as a minimum a "meta assert" with some explanation.

Reply:
ok